src/test/DoS_tests.cpp

```diff
@@ -127,7 +127,7 @@ BOOST_AUTO_TEST_CASE(stale_tip_peer_management)
         BOOST_CHECK(node->fDisconnect == false);
     }
 
-    SetMockTime(GetTime() + 3*consensusParams.nPowTargetSpacing + 1);
+    SetMockTime(GetTime() + 3*consensusParams.nPowTargetSpacing*4 + 1);
```

Comment:
Hmm... It's more like "3 blocks behind" kind of thing imo https://github.com/dashpay/dash/pull/3113/commits/f544e6a6422122020eb1ceece18251482dcd8a12#diff-eff7adeaec73a769788bb78858815c91R465, we should probably adjust `STALE_CHECK_INTERVAL` https://github.com/dashpay/dash/pull/3113/commits/f544e6a6422122020eb1ceece18251482dcd8a12#diff-410c7e6e1844912cc2d40abc4d288982R34 (bringing it down to 2.5 minutes) instead

Reply:
Hmm not sure...I think sticking with 10 minutes should be fine. But at the same time `nPowTargetSpacing * 3` is probably not enough...it happens very often that we have > 10 minutes between blocks...

Reply:
https://github.com/bitcoin/bitcoin/pull/11560#discussion_r147777589

Reply:
30 minutes would be fine, but `nPowTargetSpacing * 3` gives 7.5 minutes. What I mean is, that I maybe should add the `*4` to TipMayBeStale as well?

Reply:
Same can be said about bitcoin - it's not that uncommon for blocks to take 30+ minutes. I mean, the logic there is basically `there should be 3 new blocks by now and we have none - probably stale`. `STALE_CHECK_INTERVAL = 10` is basically `check periodically, more or less when new block is expected` imo. To preserve this logic we should adjust `STALE_CHECK_INTERVAL` to match 1 block time span. If we do NOT want to actually care about number of blocks we could just use `STALE_CHECK_INTERVAL` instead of `consensusParams.nPowTargetSpacing*4` in both cases i.e. the new logic would be `we checked 3 times and no new blocks were seen - probably stale`.

Reply:
Hmm ok convinced, I'll adjust `STALE_CHECK_INTERVAL` and push later

Reply:
done